packages/dcos-integration-test/extra/test_dcos_log.py

```diff
@@ -71,5 +72,145 @@ def test_log_proxy(cluster):
+            assert 'id' in task, 'Missing field `id` in task. Got {}'.format(state_response_json)
+            if not task['id'].startswith(task_name):
+                continue
+
+            assert 'framework_id' in task, 'Missing `framework_id` in task. Got {}'.format(state_response_json)
+            assert 'executor_id' in task, 'Missing `executor_id` in task. Got {}'.format(state_response_json)
+            assert 'id' in task, 'Missing `id` in task. Got {}'.format(state_response_json)
+            assert 'slave_id' in task, 'Missing `slave_id` in task. Got {}'.format(state_response_json)
+
+            framework_id = task['framework_id']
+            # if task['executor_id'] is empty, we should use task['id']
+            executor_id = task['executor_id']
+            if not executor_id:
+                executor_id = task['id']
+            slave_id = task['slave_id']
+
+            statuses = task.get('statuses')
+            assert isinstance(statuses, list), 'Invalid field `statuses`. Got {}'.format(state_response_json)
+            assert len(statuses) == 1, 'Must have only one status TASK_RUNNING. Got {}'.format(state_response_json)
+            status = statuses[0]
+            container_status = status.get('container_status')
+            assert container_status
+            container_id_field = container_status.get('container_id')
+            assert container_id_field
+
+            # traverse nested container_id fields
+            container_ids = []
+            while True:
+                value = container_id_field.get('value')
+                assert value
+                container_ids.append(value)
+
+                if 'parent' not in container_id_field:
+                    break
+
+                container_id_field = container_id_field['parent']
+            container_id = '.'.join(reversed(container_ids))
+            assert container_id
+
+    # validate all required fields
+    assert slave_id, 'Missing slave_id'
+    assert framework_id, 'Missing framework_id'
+    assert executor_id, 'Missing executor_id'
+    assert container_id, 'Missing container_id'
+
+    endpoint_type = 'range'
+    if stream:
+        endpoint_type = 'stream'
+    return '/system/v1/agent/{}/logs/v1/{}/framework/{}/executor/{}/container/{}'.format(slave_id, endpoint_type,
+                                                                                         framework_id, executor_id,
+                                                                                         container_id)
+
+
+def test_pod_logs(cluster):
+    test_uuid = uuid.uuid4().hex
+
+    pod_id = 'integration-test-pod-logs-{}'.format(test_uuid)
+
+    pod_definition = {
+        'id': '/{}'.format(pod_id),
+        'scaling': {'kind': 'fixed', 'instances': 1},
+        'containers': [
+            {
+                'name': 'sleep1',
+                'exec': {'command': {'shell': 'echo STDOUT_LOG;echo STDERR_LOG >&2;sleep 10000'}},
```

Comment:
Having apps (even trivial ones) without health checks makes for weird deploy conditions. I think we should update this to be like https://github.com/dcos/dcos/blob/master/packages/dcos-integration-test/extra/test_applications.py#L80

The bigger problem here is that the deploy_pod method in test_util marathon should be health check aware which it currently isnt.